source/js/main-page-controller.js

```diff
@@ -65,62 +69,78 @@ export class MainPageController {
     }
 
     // Toolbar buttons
-    let textTool = document
-      .querySelector('tool-bar')
-      .shadowRoot.querySelector('#text-tool')
+    let textTool = this.toolbar.shadowRoot.querySelector('#text-tool')
 
     textTool.addEventListener('click', (event) => {
       selectTool(textTool)
 
-      let dragView = document.querySelector('drag-view')
-      console.log(dragView)
-      console.log(dragView.textOnClick)
-      dragView.textOnClick = dragView.textOnClick ? false : true
+      console.log(this.dragview)
+      console.log(this.dragview.textOnClick)
+      this.dragview.textOnClick = this.dragview.textOnClick ? false : true
     })
     //Image Tool
-    let imageTool = document
-      .querySelector('tool-bar')
-      .shadowRoot.querySelector('#image-tool')
-
+    let imageTool = this.toolbar.shadowRoot.querySelector('#image-tool')
+    this.imageInput.addEventListener('change', (e) => {
+      const img = new Image()
+      img.src = URL.createObjectURL(e.target.files[0])
+      this.dragview.addDraggableImage({ left: '20px', top: '50px' }, img)
```

Comment:
What about image resizability? Seems like would be issue if a large image was inserted

Reply:
Working on that

Reply:
But yeah you're very right, and also might need a delete button, both should be doable

Reply:
img resize is working, but there's no ui for it. For now just drag the right bottom corner to resize anything in the  main page. I'll create a issue for it we can do it later if we have time.

Reply:
We can resize image and text box buy dragging the left bottom corner. We can add styles and resize frame for these later if we have time. I added an issue for this.